naga/src/front/wgsl/parse/conv.rs

```diff
@@ -255,6 +255,8 @@ pub fn map_standard_fun(word: &str) -> Option<crate::MathFunction> {
         "unpack2x16float" => Mf::Unpack2x16float,
         "unpack4xI8" => Mf::Unpack4xI8,
         "unpack4xU8" => Mf::Unpack4xU8,
+        // FIXME: move to correct category
+        "quantizeToF16" => Mf::QuantizeToF16,
```

Comment:
unsure which category to add this under (if that wasn't obvious from the FIXMEs :smile: )

Reply:
`computational` looks like the best fit to me. Arguably they are all computing things so, `computational` is more like the misc bucket.

Reply:
Sounds good thanks